website/templates/emails/reviews_submission_confirmation.html.mako

```diff
@@ -9,86 +9,101 @@
     %>
     <tr>
         <td>
-            <div style="margin: 40px; background: white;">
-                <p>Hello ${user.fullname},</p>
-                % if is_creator:
-                    <p>
-                        Your ${document_type}
-                        <a href="${reviewable.absolute_url}">${reviewable.title}</a>
-                        has been successfully submitted to ${reviewable.provider.name}.
+            % if document_type == 'registration':
+                Hello ${user.fullname},
+                <br>
+                Your registration has been successfully submitted to ${reviewable.provider.name}.
```

Comment:
The document has `Your registration <a href="$reviewable.absolute_url">${reviewable.title}</a> has been succes…` etc.